docs/KubeConformance-1.15.md

```diff
@@ -0,0 +1,1475 @@
+
+Ensure that scheduler accounts node resources correctly
+and respects pods' resource requirements during scheduling.
+
+
+
+## [scheduler-node-selector-not-matching](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e/scheduling/predicates.go#L351)
+
+### Release 
+Test Nodes does not have any label, hence it should be impossible to schedule Pod with
+nonempty Selector set.
+
+Ensure that scheduler respects the NodeSelector field of
+PodSpec during scheduling (when it does not match any node).
+
+
+
+## [scheduler-node-selector-matching](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e/scheduling/predicates.go#L374)
+
+### Release 
+Ensure that scheduler respects the NodeSelector field
+of PodSpec during scheduling (when it matches).
+
+
+
+## [Kubelet when scheduling a busybox command in a pod it should print the output to logs](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/kubelet_test.go#L42)
+
+### Release 
+
+
+
+## [Kubelet when scheduling a read only busybox container it should not write to root filesystem](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/kubelet_test.go#L167)
+
+### Release 
+
+
+
+## [Container Lifecycle Hook when create a pod with lifecycle hook should execute poststart exec hook properly](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/lifecycle_hook_test.go#L87)
+
+### Release 
+
+
+
+## [Container Lifecycle Hook when create a pod with lifecycle hook should execute prestop exec hook properly](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/lifecycle_hook_test.go#L98)
+
+### Release 
+
+
+
+## [Container Lifecycle Hook when create a pod with lifecycle hook should execute poststart http hook properly](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/lifecycle_hook_test.go#L109)
+
+### Release 
+
+
+
+## [Container Lifecycle Hook when create a pod with lifecycle hook should execute prestop http hook properly](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/lifecycle_hook_test.go#L122)
+
+### Release 
+
+
+
+## [MirrorPod when create a mirror pod  should be updated when static pod updated](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/mirror_pod_test.go#L60)
+
+### Release 
+
+
+
+## [MirrorPod when create a mirror pod  should be recreated when mirror pod gracefully deleted](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/mirror_pod_test.go#L82)
+
+### Release 
+
+
+
+## [MirrorPod when create a mirror pod  should be recreated when mirror pod forcibly deleted](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/mirror_pod_test.go#L97)
+
+### Release 
+
+
+
+## [Container Runtime Conformance Test container runtime conformance blackbox test when starting a container that exits it should run with the expected status](https://github.com/kubernetes/kubernetes/tree/v1.15.0/test/e2e_node/runtime_conformance_test.go#L49)
```

Comment:
This is not a conformance test; we shouldn't be pulling in from `test/e2e_node`, but more than that, the walker shouldn't be picking this up (there is no `[Conformance]` or `ConformanceIt`)

Follow the link https://github.com/kubernetes/kubernetes/blob/v1.15.0/test/e2e_node/runtime_conformance_test.go#L49 and you'll see it's pointed at
```
			"email": "image-pulling@authenticated-image-pulling.iam.gserviceaccount.com"
```

Reply:
Yeah I'm not sure where/how you're getting these.  I don't pick up anything in e2e_node and I get 215 tests instead of 177 for v1.15.0
```
spiffxp@spiffxp-macbookpro:kubernetes ((v1.15.0) %)$ go run test/conformance/walk.go --conformance test/e2e_node | tail -n2
Total Conformance Tests: 0, total legacy tests that need conversion: 0, while total tests that need comment sections: 0

spiffxp@spiffxp-macbookpro:kubernetes ((v1.15.0) %)$ go run test/conformance/walk.go --conformance test/e2e | tail -n2
Total Conformance Tests: 215, total legacy tests that need conversion: 0, while total tests that need comment sections: 0
```